### PR TITLE
Sam branch

### PR DESCRIPTION
**Pull request by sgha229**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,4 +6,4 @@ gem 'jekyll-sitemap'
 
 gem 'jekyll-assets'
 gem 'jekyll-redirect-from'
-
+gem 'bootstrap', '~> 4.4.1'
```
Comment (raney24) [inline Gemfile]: @sgha229 What was this added for? Is this PR the migration to BS4 as well?

File: internships.html [hunk 2]
```diff
@@ -79,21 +79,32 @@ <h2 class="heading-title">Find the right team for you</h2>
...
+  <div class="modal fade" id="lgx-modal-video" role="dialog">
+    <div class="modal-dialog">
+    
+      <!-- Modal content-->
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal">&times;</button>
+        </div>
+        <div class="modal-body">
+         <iframe width="575" height="450"  src="https://www.youtube.com/embed/Xhs6weqDvfg" allowfullscreen></iframe>
```
Comment (raney24) [inline internships.html]: @sgha229 Can we make this `width` responsive? Probably needs to be done in a `.scss` file. I would suggest something around 70-80% width.

Comment (raney24) [reply internships.html]: In addition, `height` can be responsive as well

File: workspace.html [hunk 3]
```diff
@@ -52,40 +52,74 @@ <h3>See the Space</h3>
...
               <div id="see-space-video-modal" class="modal fade lgx-modal">
-                <div class="modal-dialog">
-                  <div class="modal-content">
-                    <div class="modal-header">
-                      <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
-                    </div>
-                    <div class="modal-body" data-embed="0YgGYadr7rk">
-                      <iframe id="modalvideo" src="https://www.youtube.com/embed/0YgGYadr7rk" allowfullscreen></iframe>
-                    </div>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```
Comment (raney24) [inline workspace.html]: @sgha229 How come we are importing BS again here?

File: workspace.html [hunk 4]
```diff
@@ -52,40 +52,74 @@ <h3>See the Space</h3>
...
+    .bs-example{
+    	margin: 545px;
+    }
+    .modal-content iframe{
+        margin: 0 auto;
+        display: block;
+    }
+</style>
+<script>
+$(document).ready(function(){
```
Comment (raney24) [inline workspace.html]: @sgha229 This video doesn't work the same as the internship page for me

File: internships.html [hunk 5]
```diff
@@ -79,21 +79,32 @@ <h2 class="heading-title">Find the right team for you</h2>
...
-                  </div>
-                  <div class="modal-body">
-                    <iframe id="modalvideo" src="https://www.youtube.com/embed/Xhs6weqDvfg" allowfullscreen></iframe>
-                  </div>
-                </div>
-              </div>
-            </div> <!-- //.Video-->
+            <body>
+
+<div class="container">
```
Comment (raney24) [inline internships.html]: @sgha229 Great job on getting these all to work!

File: Gemfile (hunk 1 above)
Comment (sgha229) [reply Gemfile]: @raney24 Yes, this is part of the BS4 migration. I had planned to just release this alongside the other changes as well. Is this an issue?

File: internships.html (hunk 2 above)
Comment (sgha229) [reply internships.html]: Sounds good, I'll get to fixing them up.

File: workspace.html (hunk 4 above)
Comment (sgha229) [reply workspace.html]: The issue I had with this page was the format and tools I used for the internship page do not seem to work here for some reason. I'll look into making them all uniform.

File: workspace.html (hunk 3 above)
Comment (sgha229) [reply workspace.html]: That was just bad coding on my part when I was first playing around with BS.

File: workspace.html (hunk 4 above)
Comment (sgha229) [reply workspace.html]: I've managed to make the video look like the internships page. The issue with the empty YouTube thumbnails that are still appearing at the end of the modal no matter what. Do you have any insight on how to get rid of these pesky things?

Comment (sgha229) [reply workspace.html]: Never mind, I fixed it. Modal container is a wide size and shape on the page for some reason, poking around the css and js to figure out why.

**Pull request merged**